### PR TITLE
Add a ToC with links to all READMEs

### PR DESCRIPTION
**Pull request by pudelkoM**

### CODE REVIEW

**Comment by Yi-Tseng** [comment]
We also have some articles here: https://github.com/stratum/stratum/wiki
The IDE setup one might be useful and maybe we can be list in the README?

**Comment by pudelkoM** [comment]
> We also have some articles here: https://github.com/stratum/stratum/wiki
> The IDE setup one might be useful and maybe we can be list in the README?

Very good point! Yes, they should go in as well.

File: README.md [hunk 1]
```diff
@@ -23,37 +23,52 @@ Build status (master): [![CircleCI](https://circleci.com/gh/stratum/stratum/tree
...
+- [Stratum-enabled Mininet](/tools/mininet/README.md)
+- [P4Runtime write request replay tool](/stratum/tools/stratum-replay/README.md)
+
+### Other Documentation
+
+- [Installation guide: ONOS + Stratum + Barefoot Tofino switch](/stratum/docs/setup_guide_barefoot_tofino_onos.md)
+- [Trellis+Stratum example](/tools/mininet/examples/trellis/README.md)
+- [Community Channels](https://github.com/stratum/stratum/wiki/Discuss)
+- [Talks](https://github.com/stratum/stratum/wiki/Talks)
+- [testvectors](stratum/public/proto/tv.proto)
```
Comment (Yi-Tseng) [inline README.md:61]: This is the protobuf, maybe use this?
[](https://github.com/stratum/testvectors/blob/master/docs/testvectors_overview.md)

Comment (bocon13) [reply README.md:61]:
```suggestion
- [Test Vectors](https://github.com/stratum/testvectors/blob/master/docs/testvectors_overview.md)
```

File: README.md [hunk 2]
```diff
@@ -23,37 +23,52 @@ Build status (master): [![CircleCI](https://circleci.com/gh/stratum/stratum/tree
...
+- [Broadcom](/stratum/hal/bin/bcm/standalone/README.md)
+  - [How to Add a Dataplane Feature to Stratum (BCM)](/stratum/docs/extenting_stratum_bcm.md)
+- [Intel NP4](/stratum/hal/bin/np4intel/docker/README.md)
+- [bmv2](/stratum/hal/bin/bmv2/README.md)
+- [Dummy Switch](/stratum/hal/bin/dummy/README.md)
+
+### Tools
+
+- [gNMI tool](/stratum/tools/gnmi/README.md)
+- [Stratum-enabled Mininet](/tools/mininet/README.md)
```
Comment (bocon13) [inline README.md:52]:
```suggestion
- [Stratum-Enabled Mininet](/tools/mininet/README.md)
```

File: README.md [hunk 3]
```diff
@@ -23,37 +23,52 @@ Build status (master): [![CircleCI](https://circleci.com/gh/stratum/stratum/tree
...
+  - [Building Stratum for Barefoot Tofino based switches](/stratum/hal/bin/barefoot/README.build.md)
+- [Broadcom](/stratum/hal/bin/bcm/standalone/README.md)
+  - [How to Add a Dataplane Feature to Stratum (BCM)](/stratum/docs/extenting_stratum_bcm.md)
+- [Intel NP4](/stratum/hal/bin/np4intel/docker/README.md)
+- [bmv2](/stratum/hal/bin/bmv2/README.md)
+- [Dummy Switch](/stratum/hal/bin/dummy/README.md)
+
+### Tools
+
+- [gNMI tool](/stratum/tools/gnmi/README.md)
```
Comment (bocon13) [inline README.md:51]:
```suggestion
- [gNMI CLI Tool](/stratum/tools/gnmi/README.md)
```

File: README.md [hunk 4]
```diff
@@ -23,37 +23,52 @@ Build status (master): [![CircleCI](https://circleci.com/gh/stratum/stratum/tree
...
+- [Stratum-enabled Mininet](/tools/mininet/README.md)
+- [P4Runtime write request replay tool](/stratum/tools/stratum-replay/README.md)
+
+### Other Documentation
+
+- [Installation guide: ONOS + Stratum + Barefoot Tofino switch](/stratum/docs/setup_guide_barefoot_tofino_onos.md)
+- [Trellis+Stratum example](/tools/mininet/examples/trellis/README.md)
```
Comment (bocon13) [inline README.md:58]:
```suggestion
- [Trellis + Stratum example](/tools/mininet/examples/trellis/README.md)
```

File: README.md [hunk 5]
```diff
@@ -23,37 +23,52 @@ Build status (master): [![CircleCI](https://circleci.com/gh/stratum/stratum/tree
...
+- [testvectors](stratum/public/proto/tv.proto)
+- [Stratum example pipelines](/stratum/pipelines/README.md)
+  - [Main FPM pipeline](/stratum/pipelines/main/README.md)
+- [Supported gNMI paths](/stratum/docs/gnmi/supported-paths.md)
+- [Config Monitoring Service](/stratum/docs/gnmi/README.md)
+- [Transponder Abstraction Interface (TAI)](/stratum/docs/tai/README.md)
+- [P4C backend](/stratum/p4c_backends/README.md)
+- [CDLang gNMI Compliance Definition Language](/stratum/testing/cdlang/g3doc/cdlang.md)
+- [gNMI/OpenConfig Contract Definition Language tools](/stratum/testing/cdlang/README.md)
+- [gNMI test scenarios](/stratum/testing/scenarios/README.md)
```
Comment (bocon13) [inline README.md:70]:
```suggestion
- [gNMI Test Scenarios](/stratum/testing/scenarios/README.md)
```

File: README.md [hunk 6]
```diff
@@ -23,37 +23,52 @@ Build status (master): [![CircleCI](https://circleci.com/gh/stratum/stratum/tree
...
+- [SSL/TLS support and security](/stratum/lib/security/README.md)
```
Comment (bocon13) [inline README.md:71]:
```suggestion
- [SSL/TLS Support and Security](/stratum/lib/security/README.md)
```

**Review by bocon13 — APPROVED** [review]

**Pull request merged**